OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -39,7 +39,8 @@ ErrorMessageOr<void> CaptureClient::StartCapture(
     ThreadPool* thread_pool, const ProcessData& process,
```

Comment:
Bug: http://b/...

Reply:
I assume  you mean in the commit message/PR description? 

(If yes, fixed. If no, please provide more detail.)